pages/common/jhipster.md

```diff
@@ -0,0 +1,36 @@
+# jhipster
+
+> Generates a monolithic or microservices project after prompting user.
```

Comment:
Generator for creating Spring and/or Angular/React/Vue projects.
```suggestion
> Generator from creating Spring and/or Angular/React/Vue projects.
```

Reply:
I disagree with this suggestion as you can also generate a backend with Kotlin, .NET, NodeJS or Quarkus so I'd rather prefer to stay generic in the description if you don't mind?

Reply:
Hi @fdelbrayelle 

No problem, as I have no experience with JHipster my suggestion was based on the official [tech stack](https://www.jhipster.tech/tech-stack/)

Even so I think we could narrow a little bit the description, maybe focusing on the web scenario.

What do you think about "Web application generator using either monolithic or microservice architecture"

Reply:
It sounds not bad at all!